src/versioning/migrations/1.15.5.ts

```diff
@@ -0,0 +1,17 @@
+import { nestedCountersInstance } from '@shardus/core'
+// import { ShardeumFlags } from '../../shardeum/shardeumFlags'
+import { Migration } from '../types'
+import { shardusConfig } from '../..'
+
+// This has been baked into settings and is not needed, but the goal is to keep one migration as
+// an example for when we need to migrate again.
+
+export const migrate: Migration = async () => {
+  console.log('migrate 1.15.5')
+  nestedCountersInstance.countEvent('migrate', 'calling migrate 1.15.5')
+
+  shardusConfig.p2p.dropNGTByGossipEnabled = true
```

Comment:
Im wondering if this should be false.  We wouldnt want to somehow migrate nodes to the new version and have this be set to true when the default will be false until we decide to flip the flag to true on the whole network

Reply:
This runs when we update the active version, not when we let in new nodes with a different version, so I think what youre thinking is whats going to happen. I followed the template from other migration files, but I guess getting @afostr to review this as well would be best

Reply:
Yeah let’s get clarification from Andrew as we could technically set active version with out flipping this flag for whatever reason